apps/docs/src/guide/types/numbers.md

```diff
@@ -0,0 +1,29 @@
+# Numbers
+
+In Sway, there are multiple types of numbers:
+
+1. `u8` (8-bit unsigned integer)
+2. `u16` (16-bit unsigned integer)
+3. `u32` (32-bit unsigned integer)
+4. `u64` (64-bit unsigned integer)
+5. `u256` (256-bit unsigned integer)
+
+Conveniently, all of these types are handled the same way in the TS SDK. This guide explains how to create and interact with Sway numbers while using the SDK.
+
+## Creating Numbers
+
+Just like Ethereum, when you pass in a number to a Sway program from JavaScript, you must first convert it to a `BigNum` object. Here's how you can do that:
```

Comment:
Everything in this PR mentions `BigNumber`, except here.

When we mention `BigNum`, are we referring to this? 🤔 
 - https://www.npmjs.com/package/bignum

Reply:
This is because I accepted @danielbate's suggestion for one occurence of the term but didn't update the rest.

Now that I think about it, originally, I used the term `BigNumber` as a 'general' term referring to big numbers, not a term specific to any libraries.

Maybe what we can do is 

1. Still use `BigNumber` everywhere but make it link to the implementation for `bn` in our codebase
and 2. Show people how to convert from a `BigInt` (or whatever the compatible BigNumber variation for `ethers` is) for usage with the TS SDK - _if_ required. Need to investigate if it is required.

Thoughts? @danielbate @arboleya

Reply:
This was a change I recommended. We should refer to it as `BigNum` through, as that it is what `bn.js` [refer to it as](https://github.com/indutny/bn.js/blob/master/README.md?plain=1#L3). IMO `BigNumber` is referring to [bignumber.js](https://www.npmjs.com/package/bignumber.js/v/4.0.4).